firebase-auth/src/iosMain/kotlin/dev/gitlive/firebase/auth/user.kt

```diff
@@ -45,7 +45,7 @@ actual class FirebaseUser internal constructor(val ios: FIRUser) {
         ios.awaitResult<FIRUser, FIRAuthDataResult?> { reauthenticateWithCredential(credential.ios, it) }.run { Unit }
 
     actual suspend fun reauthenticateAndRetrieveData(credential: AuthCredential): AuthResult =
-        AuthResult(ios.awaitResult { reauthenticateAndRetrieveDataWithCredential(credential.ios, it) })
+        AuthResult(ios.awaitResult { reauthenticateAndRetrieveData(credential) })
```

Comment:
this looks like its calling itself - do you need to revert this back to using https://firebase.google.com/docs/reference/ios/firebaseauth/api/reference/Classes/FIRUser#-reauthenticatewithcredential:completion:

Reply:
Right, totally missed that. Fixed :)